demos/browser/app/meetingV2/video/VideoTileCollection.ts

```diff
@@ -452,10 +454,10 @@ export default class VideoTileCollection implements AudioVideoObserver {
     // We need to manually control visibility of paused tiles anyways so we just do
     // everything here, even though the preference manager adding/removing will
     // result in tile callbacks as well.
-    for (let [index, videoTile] of this.tileIndexToDemoVideoTile.entries()) {
+    for (let videoTile of this.tileIndexToDemoVideoTile.values()) {
         if (attendeesToShow.includes(videoTile.attendeeId)) {
             videoTile.show(false);
-        } else if (this.tileIndexToTileId[index] !== this.findContentTileId()) { // Always show content
```

Comment:
This breaks because of the local content that automatically makes a tile. It doesn't really need to make UX decisions like this (not like UX eng ever assisted with demo) so simpler is better IMO